common/SolidityTestUtils.js

```diff
@@ -9,7 +9,7 @@ async function didContractThrow(promise) {
   return false;
 }
 
-async function minePendingTransactions(web3, time) {
+async function advanceBlockAndSetTime(web3, time) {
```

Comment:
I renamed this function as what it does is actually advance the chain by one block and set the time. Previous implementation did not export this function; it was only used internally by the `mineTransactionsAtTime` function. I renamed this and changed how it's called in `mineTransactionsAtTime` such that it can be used by other scripts, like the `VoteSimulator` that follows.

Reply:
SGTM!